libraries/dawg-show/src/main/webapp/views/simplified.jsp

```diff
@@ -158,12 +133,14 @@ String fullAudioUrlOGGExtension = audioUrl + "/play1.ogg";
                 <div id="remote" class="remoteContainer" style="width=90%;">
                     <jsp:include page="<%=stdRemotePage %>" />
                 </div>
+                <% if (!isXR11) { %>
```

Comment:
I think we should try to avoid any mentions of specific remotes in the page. Instead parameterize and set the parameter somewhere else, like the remote definition.

Like if we add another remote that we also don't want to have the mute displayed, then we have to add another if statement here, doesn't scale.